packages/manager/apps/dedicated/client/app/dedicated/dedicated-server/clusters/cluster/cluster.constant.js

```diff
@@ -0,0 +1,4 @@
+export const STATUS_DONE = 'DONE';
```

Comment:
no other values for status ?
```suggestion
export const STATUS = { DONE: 'DONE' };
```